src/Wares/SilverOre.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Hyperdrive\Wares;
+
+class SilverOre extends Commodity
+{
+    public function __construct(int $quantity)
+    {
+        parent::__construct($quantity);
+
+        $this->weightPerUnit = 10;
+    }
```

Comment:
Czy nie możemy po prostu zainicjalizować `$wagePerUnit` w taki sposób?
```suggestion
    protected int $weightPerUnit = 10;
```